CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 
 ### Changed
 - update `@deltachat/stdio-rpc-server` and `deltachat/jsonrpc-client` to `2.12.0`
+- Add some docstrings for `useMessageList` hook, `getDirection` function, `copyToBlobDir` function and `locations.ts` all in frontend package
```

Comment:
I would not mention that in the changelog. It is only for devs and even they not "neccessarily" need to know what has changed

use #skip-changelog instead